packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -19,6 +20,15 @@ const propTypes = {
   */
   helpItems: PropTypes.array,
   /**
+  * Alert banner
+  */
+  siteAlert: PropTypes.arrayOf(
```

Comment:
I don't think my comments were clear, but this should just be a react component. The consumer of the layout can pass in either a single alert or an array of alerts or nothing. All the layout does is render it in the right spot.